src/sql/workbench/contrib/searchViewPane/browser/searchWidget/simpleSearchWidget.ts

```diff
@@ -102,21 +109,32 @@ export class NotebookSearchWidget extends Widget {
 		});
 		this.accessibilityService.onDidChangeScreenReaderOptimized(() => this.updateAccessibilitySupport());
 		this.updateAccessibilitySupport();
+
+		this._register(this.onSearchSubmit(ops => this.triggerQueryChange(ops)));
+		this._register(this.onSearchCancel(({ focus }) => this.cancelSearch(focus)));
+		this._register(this.searchInput.onDidOptionChange(() => this.triggerQueryChange()));
+
+
+		this.trackInputBox(this.searchInputFocusTracker);
+	}
+
+	public get searchView(): ISearchResultsView | undefined {
+		return <NotebookSearchResultsView>this.parentContainer.getView(NotebookSearchResultsView.ID) ?? undefined;
```

Comment:
This should be generic - not tied to Notebooks view

Reply:
Make sure you go through all these generic component files and scrub them for any references to Notebook, I wouldn't expect them to have any left after you're done.